### PR TITLE
chore: Make redis, postgres and rabbitmq hosts configurable via env variables

### PR DESCRIPTION
**Pull request by lotooo**
I am trying to deploy our musicbrainz mirror in kubernetes using an external DB.

This PR updates the config files to read the db,redis,mq hosts from env variables and default them to the current value

### CODE REVIEW

**Review by yvanzo — CHANGES_REQUESTED** [review]
Thank you for contributing to this repository.

The general idea is great.

I pushed a few amending commits to replace the last occurrences of `db` and `redis` in build files.

What is still missing:

1. `depends_on` still refer to service names in compose files; See:
   ```sh
   git grep -Pw -- '- (db|mq|redis)' '**yml'
   ```
2. Many occurences of `db`,  `mq` and `redis` remain in helper scripts under the directory `admin/`; See:
   ```sh
   git grep -Pw 'db|mq|redis' admin/
   ```
3. New variables have to be documented in `README.md`

For 1. I have no idea how to handle it or if it even matters when using Kubernetes.
For 2. & 3. I can do if needed whenever I have time but that isn’t my current priority.

**Comment by lotooo** [comment]
Thanks for the feedback !

1. doesn't need be changed as these stay the default values so it's fine to let them exist with the static name in the docker-compsoe file.

3. I update the README.

Working on the 2. now.

**Comment by lotooo** [comment]
For 2, I read those scripts (I just checked the one in the output of `git grep -Pw 'db|mq|redis' admin/`) and all of them are meant to be used locally and connect using docker-compose so there shouldn't have any impact (as the hosts name and the depends_on is not changed in the docker-compose.yml).

tldr; I don't think any of them has to be changed as they are not included in the Docker image and not meant to be executed from inside any of the containers.

**Review by yvanzo — APPROVED** [review]
Thanks again for your contribution!

I improved the documentation you provided (for 3.) by mentioning that:
* the unused services will still be running (for 1.),
* some (dispensable) helper scripts will no longer work (for 2.),
* (potentially) some actions can be needed when switching to a new back-end server.
